src/webview/SampleView.ts

```diff
@@ -135,14 +136,34 @@ export class SampleView extends View {
     }
 
     private async getSamplePage(options: any): Promise<any> {
-
+        let validInternetConnection: boolean = true;
+        const outputAdapter: VSCodeBlockchainOutputAdapter = VSCodeBlockchainOutputAdapter.instance();
         const templatePath: string = path.join(__dirname, '..', '..', '..', 'templates', 'SampleView.ejs');
 
         const converter: showdown.Converter = new showdown.Converter();
-        const response: any = await Axios.get(options.sample.readme);
-        const text: string = response.data;
-        const html: string = converter.makeHtml(text);
-        options.html = html;
+        const repositoryRegistry: RepositoryRegistry = RepositoryRegistry.instance();
```

Comment:
you don't need to do this unless you don't have an internet connection

Reply:
Fixed - moved it into the if statement once it checks that there is no internet connection.